### ISSUE TITLE
bug: washed out video player colors 

### ISSUE THREAD

**Issue by classicrocker883** [ReVanced/revanced-patches#1932]
### Application

Youtube Revanced using Windows 11 through WSA (Windows Subsystem for Android).
I was able to install the APK using the typical method when you search how to. 

### Issue

The App works fine being used IN Windows, however watching Videos have a grey tint. it is like contrast is very low. Overall, all the thumbnails and UI appear normal, it is just the Box/window where the video is being played.

I tried different settings like on/off ambiant mode, or Auto HDR brightness, to no avail. 

### Patch

I would like the video player to have this issue fixed. Perhaps it is a bug, or perhaps there can be a way to change video display properties, such as brightness, color, contrast, ect...

maybe the issue stems from HDR brightness? the image is too grey like "greyed-out". there is no black color - it's not that deep however everywhere else in the app appears normal. this is Only the video playing. 

### Motivation

because I would like to use this while on a Windows desktop PC. it would be great to view videos through my computer - as well as my android phone - without Ads popping up and what not. 

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Check on unpatched app.

**Comment by classicrocker883** [comment]
I haven't been able to install the Extended version in Windows. it does install, but upon opening it opens a blank window then quickly closes(terminates).

the General YouTube Revanced worked fine, however that had a couple bugs like pressing the Library button would cause it to close/terminate.

no real issue using Samsung Galaxy phone.

care to point out or explain the "unpatched app" for me?

**Comment by classicrocker883** [comment]
@oSumAtrIX way to lock the issue before I was done getting answers with the other issue. if you're so proud of what you're working on how about make it easy for someone like me to use it? 

you want people using this right? you left me in the dust, shut me out while I was lost. thanks for literally nothing dude. 

honestly then what is the point of all these repositories, the site, and Revanced if I can even use it? instead I have to rely on some other party?

am I supposed to build my own APK? like what am I supposed to do now? how do I use Revanced on my phone or PC as intended?

wish there was some guide. not everyone is some super computer guy and knows what each thing does. where's the "That WAS Easy" button?

you however aren't making it easy for anyone.

you were vauge with your answers and did not help at all.

**Comment by oSumAtrIX** [comment]
I directly answered your questions, abusing an issue in the wrong place; we have discussions and plenty of resources in our places. Anyways offtopic. Please refrain from abusing GitHub issues.

**Comment by classicrocker883** [comment]
@oSumAtrIX we have different meanings for the word abuse. in English it has negative connotation and is an insult to my intelligence to suggest I "abuse" anything at all.

to me, I have a legitimate issue using this platform. if it's a different APK or not directly attributed to this repository in particular then that is no fault to mine. wherever I used the Application, it is called Revanced, and upon searching, even linking from the site where I downloaded the App, it gave this github repository. Anyone will conclude that this is the right place to address my issue. 

it seems like a mistake to you, that is no fault to me for you thinking that. As far as I know, I am in the right place.

If it is a problem with you, you can at least be kind enough to guide me to the correct place, since clearly I am confused why the App I used and this repo aren't related.

and if this is in charge of making things work for the better, then by all means address the issue instead of leaving me to guess and be lost.

by no means am I willing to use and "abuse" an issues for github. clearly you understand that, you don't have to be a jerk by saying I am doing something wrong because it doesn't compute with your typical input.

anyway, in the future if someone needs help with an issue, don't shut them out. wait until they aren't satisfied. that's how we do things in America.

**Comment by oSumAtrIX** [comment]
TLDR, this is too long, i am not reading all of that.
